inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -319,24 +319,53 @@ void SubstituteSoftSignPass::run() {
         }
         if (cont) continue;
         if (!hasNChildren(abs, 1)) continue;
-        auto power = getNthChild(abs, 0);
+        auto addition = getNthChild(abs, 0);
+        InferenceEngine::CNNLayerPtr power = nullptr;
+
+        if (!LayerInfo(addition).isPower()) continue;
+        auto powerLayer = LayerInfo(addition).as<PowerLayer*>();
+        if (powerLayer->power != -1) {
+            //if first one is power equals 1, then it still can be a softsign
+            //with two power operations, one for add and one for power
+            if (powerLayer->power != 1) {
+                continue;
+            } else {
+                if (powerLayer->offset != 1) continue;
+                if (powerLayer->scale != 1) continue;
+
+                power = getNthChild(addition, 0);
+                if (!LayerInfo(power).isPower()) continue;
+                auto powerLayer_1 = LayerInfo(power).as<PowerLayer*>();
+                if (powerLayer_1->power != -1) continue;
+                if (powerLayer_1->offset != 0) continue;
```

Comment:
not explained, why child power offset must be 0 while original power offset must be 1?

Reply:
Softsign is a computation that equals x/(1+|x|). Power operation used here is an operation which calculates: scale * x ^ power + offset [(here)](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Subgraph_Replacement_Model_Optimizer.html). When addition is added separately, offset is set to 0.